### PR TITLE
Simplify buttons sets

### PR DESCRIPTION
**Pull request by dgdavid**
As part of the [menu-driven interface](https://trello.com/c/NITXJRBE/2055-menu-driven-interface-finish-it), buttons below devices tables have been simplified by getting rid of _menu-buttons_ and just displaying simple buttons to trigger the most common actions. The rest of them are going to be available through the menu.

E.g., from

Screenshot_20200921_082253

to

Screenshot_20200921_082217

### CODE REVIEW

File: src/lib/y2partitioner/widgets/btrfs_modify_button.rb [hunk 1]
```diff
@@ -18,45 +18,31 @@
...
+require "y2partitioner/widgets/device_button"
 require "y2partitioner/actions/edit_btrfs"
-require "y2partitioner/actions/edit_btrfs_devices"
 
 module Y2Partitioner
   module Widgets
-    # Menu button for modifying a Btrfs filesystem
-    class BtrfsModifyButton < DeviceMenuButton
+    # Device button for modifying a Btrfs filesystem
+    class BtrfsModifyButton < DeviceButton
```
Comment (ancorgs) [inline src/lib/y2partitioner/widgets/btrfs_modify_button.rb:27]: This name is now inconsistent with the rest. It should be `BtrfsEditButton`. There are no other `WhateverModifyButton` any longer. Isn't it?

File: test/y2partitioner/widgets/device_buttons_set_test.rb [hunk 2]
```diff
@@ -122,14 +123,14 @@
       let(:scenario) { "bcache2.xml" }
       let(:device) { device_graph.find_by_name("/dev/bcache0") }
 
-      it "replaces the content with buttons to modify, to delete and to manage partitions" do
+      it "replaces the content with buttons to modify, to delete and to adda a partition" do
```
Comment (ancorgs) [inline test/y2partitioner/widgets/device_buttons_set_test.rb:126]: typo: adda

File: src/lib/y2partitioner/widgets/btrfs_modify_button.rb (hunk 1 above)
Comment (dgdavid) [reply src/lib/y2partitioner/widgets/btrfs_modify_button.rb:27]: It is

Comment (dgdavid) [reply src/lib/y2partitioner/widgets/btrfs_modify_button.rb:27]: In fact, there is already a [`BtrfsEditButton`](https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2partitioner/widgets/btrfs_edit_button.rb). Nice catch :)

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**